templates/getting_started_typescript/src/main.ts

```diff
@@ -0,0 +1,33 @@
+// This is the main Node.js source code file of your actor.
+// An actor is a program that takes an input and produces an output.
+
+// For more information, see https://sdk.apify.com/
+import Apify from 'apify';
+
+interface InputSchema {
+    firstNumber: number;
+    secondNumber: number;
+}
+
+Apify.main(async () => {
+    console.log('Loading input')
+    // Structure of input is defined in INPUT_SCHEMA.json.
+    const input = await Apify.getInput() as InputSchema;
+    console.log('First number: ', input.firstNumber);
+    console.log('Second number: ', input.secondNumber);
+
+    // 👉 Complete the code so that result is
+    // the sum of firstNumber and secondNumber.
+    // 👇👇👇👇👇👇👇👇👇👇
+    const result = ;
```

Comment:
what about keeping the code valid by setting the default to `0`?
```suggestion
    const result = 0;
```

Reply:
Dunno, I wanted the run to fail if they don't fill in something, instead of showing wrong sum in the output.